### PR TITLE
update_excercises changed to use filepath instead of projects_dir

### LINKED ISSUE #165: Update_exercises is currently only for projects_dir

### PR DESCRIPTION
**Pull request by KalliMiika**
resolves #165

### CODE REVIEW

**Comment by KalliMiika** [comment]
resolves #165

File: tmc-langs/src/lib.rs [hunk 1]
```diff
@@ -520,32 +520,42 @@ pub fn init_tmc_client_with_credentials(
 // TODO: parallel downloads
 pub fn update_exercises(
     client: &TmcClient,
-    client_name: &str,
+    projects_dir: &Path,
 ) -> Result<DownloadOrUpdateCourseExercisesResult, LangsError> {
-    log::debug!("updating exercises for {}", client_name);
+    log::debug!("updating exercises in {}", &projects_dir.display());
```
Comment (Heliozoa) [inline tmc-langs/src/lib.rs]: The & here is unnecessary

**Review by Heliozoa — CHANGES_REQUESTED** [review]
Still contains debug prints and an unwrap, looks fine otherwise

File: tmc-langs/src/lib.rs (hunk 1 above)
Comment (KalliMiika) [reply tmc-langs/src/lib.rs]: removed debug prints and fixed the unwrap

**Review by Heliozoa — APPROVED** [review]

**Pull request merged**